app/build.gradle

```diff
@@ -45,6 +45,8 @@ dependencies {
     // unit tests
     testCompile 'junit:junit:4.12'
     compile "org.jetbrains.kotlin:kotlin-stdlib-jre7:$kotlin_version"
+    // for Google Mobile Vision Text API
+    compile 'com.google.android.gms:play-services-vision:9.0.0+'
```

Comment:
put the exact version number instead of 9.0.0+, so that we have more control over which version we're using